### PR TITLE
BitSum.Coreprio version 0.0.6.8

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/29078)

### CODE REVIEW

**Comment by BajajAditya123** [comment]
@ItzLevvie now it is fine

**Comment by BajajAditya123** [comment]
what?

**Comment by BajajAditya123** [comment]
Lets retry

**Comment by BajajAditya123** [comment]
oh ok

**Comment by BajajAditya123** [comment]
@ItzLevvie what should I do?

**Comment by Trenly** [comment]
> @ItzLevvie what should I do?

I believe you need to remove the `LLC` from this file name
image

**Comment by BajajAditya123** [comment]
done

**Comment by BajajAditya123** [comment]
Thanks

**Comment by BajajAditya123** [comment]
can any mod review my PR?

**Comment by BajajAditya123** [comment]
Are there no mods online??

**Comment by BajajAditya123** [comment]
@ItzLevvie where are you??

File: manifests/b/BitSum/Coreprio/0.0.6.8/BitSum.Coreprio.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: BitSum.Coreprio
+PackageVersion: 0.0.6.8
+PackageLocale: en-US
+Publisher: BitSum
```
Comment (ItzLevvie) [inline manifests/b/BitSum/Coreprio/0.0.6.8/BitSum.Coreprio.locale.en-US.yaml:7]:
```suggestion
Publisher: Bitsum
```
to match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by BajajAditya123** [comment]
what?

**Review by ItzLevvie — APPROVED** [review]

**Comment by BajajAditya123** [comment]
yay

**Pull request merged**